sidecars/vale/Dockerfile

```diff
@@ -28,8 +28,8 @@ RUN mkdir /projects ${HOME} && \
 
 ADD etc/entrypoint.sh entrypoint.sh
 
-# Default vale.ini is needed
-ADD etc/vale.ini ${HOME}
+# Default _vale.ini is needed
+ADD etc/_vale.ini ${HOME}
```

Comment:
I the name changed in the newest version?

Reply:
Yes, it doesn't work with the old name